catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -605,31 +605,31 @@ const WorkDetails = ({
   );
 
   return (
-    <Space
-      v={{
-        size: 'xl',
-        properties: ['padding-top', 'padding-bottom'],
-      }}
-      className={classNames({
-        row: true,
-      })}
-    >
-      <TogglesContext.Consumer>
-        {({ archivesPrototype }) =>
-          archivesPrototype && isInArchive ? (
-            <div className="container">
-              <div className="grid">
-                <Content />
-              </div>
+    <TogglesContext.Consumer>
```

Comment:
A nicer syntax to this is to use the `const { archivesPrototype } = useContext(TogglesContext)`